src/Button/Button.tsx

```diff
@@ -27,9 +27,8 @@ export const Button: React.NamedExoticComponent<ButtonProps> = React.memo((props
 
     function getButtonTheme(): string {
         switch (props.theme) {
-            case "secondary": return "outline-primary";
-            case "alternative": return "secondary";
-            case "primary": case "ghost-dark": case "ghost-light": case "anchor": case "danger": return props.theme;
+            case "anchor": return "link";
```

Comment:
we should remove anchor from here as well

Reply:
```suggestion
            case "anchor": return "link"; @deprecated
```

Reply:
I added a console.warn instead, because "anchor" is a string literal option, there's no good way to put a @deprecated mark on it. If you know a better way let me know. See commit